pkg/service/replay/service.go

```diff
@@ -32,6 +32,8 @@ type Service interface {
 	GetTestSetStatus(ctx context.Context, testRunID string, testSetID string) (models.TestSetStatus, error)
 	RunApplication(ctx context.Context, appID uint64, opts models.RunOptions) models.AppError
 	Normalize(ctx context.Context) error
+	DenoiseTestCases(ctx context.Context, testRun string, testSetID string, noiseParams []models.NoiseParams) error
```

Comment:
We may need to correct the naming here because there is normalise as well as normaliseTestcases. may cause confusion in contract.